app/controllers/candidate_interface/application_form_controller.rb

```diff
@@ -1,8 +1,17 @@
 module CandidateInterface
   class ApplicationFormController < CandidateInterfaceController
     def show
-      redirect_to candidate_interface_application_form_path if params[:token]
-      @application_form_presenter = CandidateInterface::ApplicationFormPresenter.new(current_candidate.current_application)
+      return redirect_to candidate_interface_application_form_path if params[:token]
+
+      if current_candidate.current_application.complete?
```

Comment:
The `enum` already gives us accessor methods, so you can replace this with `application_choice#application_complete?`

Reply:
Nice 👍 

I've removed the `complete?` method on the `application_choice`, but kept it on the `application_form`; makes sense?